frontend/src/App.vue

```diff
@@ -94,44 +95,95 @@ export default {
       consoleClass: "",
       logText: "",
       showConsole: false,
-      messages: {}
+      messages: {},
+      showBusyIndicator: false
     };
   },
   computed: {
-    isLoading() {
-      return !this.prompts.length || (this.currentPrompt.status === 'pending' && !this.isDone)
-    },
     isLoadingColor() {
-      const propertyValue = getComputedStyle(document.documentElement).getPropertyValue("--vscode-progressBar-background");
-      return propertyValue ? propertyValue : "#0e70c0"
+      return getComputedStyle(document.documentElement).getPropertyValue("--vscode-progressBar-background") || "#0e70c0"
     },
     selectedGeneratorHeader() {
       return this.messages.selected_generator + this.generatorName;
     },
     currentPrompt() {
-      const prompt = _.get(this.prompts, "[" + this.promptIndex +"]")
-      
-      const answers = _.get(prompt, "answers", {})
-      const questions = _.get(prompt, "questions", [])
+      return _.get(this.prompts, "[" + this.promptIndex +"]")
+    },
+    currentPromptAnswers() {
+      const answers = _.get(this.currentPrompt, "answers", {})
+      const questions = _.get(this.currentPrompt, "questions", [])
       for(const question of questions) {
         _.set(answers, [question.name], (question.isWhen === false ? undefined : question.answer))
       }
-      _.set(prompt, "answers", answers)
-      
-      return prompt
+      return answers
+    },
+    clonedAnswers() {
+      return _.cloneDeep(this.currentPromptAnswers)
     }
   },
   watch: {
-    "currentPrompt.answers": {
+    "prompts": {
+      handler(promptsVal) {
+        if (_.isEmpty(promptsVal)) {
+          this.showBusyIndicator = true
+        } else {
+          this.showBusyIndicator = false
+        }
+      }
```

Comment:
this. setBusyIndicator()
--------------------------------------------------
methods: {
    setBusyIndicator() {
      this.showBusyIndicator = !this.prompts.length || (this.currentPrompt.status === 'pending' && !this.isDone);
    },

Reply:
Done